script/index.coffee

```diff
@@ -516,13 +513,14 @@ api.wrap = (user, main=true) ->
         flags.dropsEnabled = false
         flags.classSelected = false
         flags.levelDrops = {}
-        # Award an achievement if this is their first Rebirth, or if they made it further than last time
+        # Award an achievement if this is their first Rebirth, or if they made it further than last time, or if they hit over 100
         if not (user.achievements.rebirths)
           user.achievements.rebirths = 1
           user.achievements.rebirthLevel = lvl
-        else if (lvl > user.achievements.rebirthLevel or lvl is 100)
-          user.achievements.rebirths++
+        else if (lvl > user.achievements.rebirthLevel)
           user.achievements.rebirthLevel = lvl
+        if (lvl >= 100)
```

Comment:
Way I'm reading this, if the user buys an Orb for the first time at level 100+, they will get 2 Rebirth achievements. The `if not` clause will set it to 1, then the `if (lvl >= 100)` bit increments it by one.

I would instead change `else if (lvl > user.achievements.rebirthLevel or lvl is 100)` to `else if (lvl > user.achievements.rebirthLevel or lvl >= 100)`.